tests/ansible_collections/roles/add-custom-repos/main.yml

```diff
@@ -5,4 +5,6 @@
 - include: rhel7-repos.yml
   when: ansible_facts['distribution_major_version'] == "7"
 - include: rhel8-repos.yml
-  when: ansible_facts['distribution_major_version'] == "8"
+  when: ansible_facts['distribution_version'] == "8.5" or ansible_facts['distribution_version'] == "8.6"
```

Comment:
When Oracle Linux 8.7 is released, we'll need to have in this condition only 8.5 and 8.7 (not 8.6 anymore). OL 8.7 will be supported (because it's going to be the latest minor version), and 8.6 will be supported as well however that will need RHEL 8.6 EUS custom repos. That means 8.6 will move two lines below.

Reply:
You can add a comment describing what's to be done half a year later.
```suggestion
  # When Oracle Linux 8.7 is released, the "8.6" needs to change to "8.7" and the
  # "8.6" is to be moved to the condition below related to enabling RHEL 8 EUS repos
  when: ansible_facts['distribution_version'] == "8.5" or ansible_facts['distribution_version'] == "8.6"
```

Reply:
Alright I'll update the comment